lib/transport.rb

```diff
@@ -0,0 +1,11 @@
+class Transport
+  attr_accessor :max_weight, :speed, :available
+
+  def delivery_time(distance)
+    (distance / speed.to_f).round(2)
+  end
+
+  def delivery_speed_by_weight
+    (speed.to_f / max_weight).round(2)
+  end
+end
```

Comment:
I don`t see here comparable method from task requirement?
'расширить класс Transport чтобы объекты этого класса можно было сравнивать по скорости доставки 1 кг посылки'

Reply:
Class Transport does not have required attributes from task
'создать базовый класс Transport c 3 атрибутами(max_weight, speed и available)'
your Transport object => #<Transport:0x00007fd7ae032ae0>

Reply:
Bike, Car, Transport have same attributes like :max_weight, :speed , :available. So you have to initialize it in Transport class instead of Bike and Car. Also it will wif previous comment.
Here is exaple you need from Bike class
def initialize
    super(Constants::BIKES[:max_weight], Constants::BIKES[:speed], true)
    @max_distance = Constants::BIKES[:max_distance]
  end

Reply:
Add comparable method, add required attributes.

Reply:
Change initialize method of Bike, Car with super method. Would you explain how to implement a comparable method to compare two objects? If my method compare_delivery_speed_by_weight_with is useless.

Reply:
@DavydovDenys you can do it via comparable module. It was explained in task tips.
https://ruby-doc.org/core-1.8.6/Comparable.html